app/src/main/java/com/example/vehicleinfocheck/WebScraper.java

```diff
@@ -95,4 +95,6 @@ public void setOnPageLoadedListener(onPageLoadedListener onpageloadedlistener){
     public interface onPageLoadedListener{
         void loaded(String URL);
     }
-}
\ No newline at end of file
+}
```

Comment:
47 - Set user agent to desktop method is not clear! Comments will be appreciated
73 - run2 is not a suitable method name and doesn't describe anything

Reply:
I have added the necessary comments to setUserAgentToDesktop method in the common code review branch. I am not really sure why this method (setting user agent to desktop instead of default, in this case, android device) was used in the repo I used as a reference. I tried running the app without executing the method and it worked. Should I still keep the method?